docs/modules/ROOT/pages/extensibility.adoc

```diff
@@ -79,16 +79,15 @@ They can be deployed as-is or used as templates for customization.
 
 Some presets are:
```

Comment:
The table of contents on this page has a different format (not using the **Table of Contents** heading as the rest of the pages).